### DISCUSSION TITLE
sqlalchemy_utils.create_database not working on sqlalchemy 2

### DISCUSSION THREAD

**Discussion by juancotrino** [sqlalchemy/sqlalchemy#9239] [Usage Questions] [answered]
I used this simple code in both Python `3.8.13` and `3.11.1` using sqlalchemy `2.0.1`:

```
from sqlalchemy import create_engine
from sqlalchemy_utils import database_exists, create_database

engine = create_engine("postgres://localhost/mydb")
if not database_exists(engine.url):
    create_database(engine.url)

print(database_exists(engine.url))
```

I got this error:

```
AttributeError: 'Engine' object has no attribute 'database'
```

However in both Python `3.8.13` and `3.11.1` and sqlalchemy `1.4.29` works just normal.

Thanks for your help!

**Comment by CaselIT** [comment, marked as answer]
Hi,

`create_database` comes from `sqlalchemy_utils`, not `sqlalchemy`. You should probably report it to the `sqlalchemy_utils` mantainers
